packages/rest/src/router/redirect-route.ts

```diff
@@ -37,11 +42,54 @@ export class RedirectRoute implements RouteEntry, ResolvedRoute {
     response.redirect(this.statusCode, this.targetLocation);
   }
 
+  /**
+   * handle redirection to a target location
+   *
+   * @param response
+   */
+  redirect(response: Response): void {
+    response.statusCode = this.statusCode || 302;
+    response.setHeader('Location', this.targetLocation);
+    response.setHeader('Content-Length', '0');
+    response.end();
+  }
+
   updateBindings(requestContext: RequestContext) {
     // no-op
   }
 
   describe(): string {
     return `RedirectRoute from "${this.sourcePath}" to "${this.targetLocation}"`;
   }
+
+  /**
+   * get target location
+   */
+  getTargetLocation() {
+    return this.targetLocation;
+  }
+
+  /**
+   * get status code
+   */
+  getStatusCode() {
```

Comment:
Don't introduce java-like getter here. Simply change the property to be `public readonly statusCode`.

Reply:
agreed